Makefile

```diff
@@ -21,6 +21,8 @@ TEST_OPTIONS?=
 
 OC?=oc
 
+OPERATOR_SDK_URL=https://github.com/operator-framework/operator-sdk/releases/download/v0.12.0/operator-sdk-v0.12.0-x86_64-linux-gnu
```

Comment:
I know this is WIP, but could we have the version in a separate variable so that we have a single place to change when we bump the version?